src/ProgressOnderwijsUtils/SchemaReflection/DmlTableTrigger.cs

```diff
@@ -3,14 +3,15 @@
 
 namespace ProgressOnderwijsUtils.SchemaReflection
 {
-    public sealed record DmlTableTrigger(DbObjectId ObjectId, string Name, DbObjectId TableObjectId) : IWrittenImplicitly
+    public sealed record DmlTableTrigger(DbObjectId ObjectId, string Name, DbObjectId TableObjectId, string Definition) : IWrittenImplicitly
     {
         public static DmlTableTrigger[] LoadAll(SqlConnection conn)
             => SQL($@"
                     select
                         ObjectId = tr.object_id
                         , tr.name
                         , TableObjectId = t.object_id
+                        , Definition = OBJECT_DEFINITION(tr.object_id)
                     from sys.triggers tr
                     join sys.tables t on t.object_id = tr.parent_id

```

Comment:
deze join is toch overbodig?

Reply:
je zou ook nog een trigger op een view kunnen hebben en dan is t.object_id niet een TableObjectId

Reply:
Precies dat ja.

Reply:
Maakt dat uit?

Reply:
Alleen in zoverre, dat ik nu de triggers aan de tabellen hang.